tutorials/robot-marbles-part-7/robot-marbles-part-7.ipynb

```diff
@@ -0,0 +1,653 @@
mCW9JtKtVguF7/8h7DMGHpYU0NZN0RA6NjaWojZdCY+Jkh+z7jiEJX3MAFh6bAubrxyhSN78XLVep3AMAHP91rLj+nEW95pAn3rJ79S0O+TFzAOr5a+L5M3POztfNDQ0kj1fUV5/+UDPVdaUzFeYvUZL0FT/9b4SiYROKyZxKuAGAHpaOuw3dlDKajEj7L19hiFec4hOmLn0rdeWnWMWZmsJRiqVsvHyIWz2rSQ4IpTxLXuxsIdRunYP/+nkJvcsYIbvKnxun8Gl3xT0a7dU+Prg8BD+dZa5MNl2GZFsXTQg6DWTd7tw9OFlapaoyMnJbrT7J7ENdub+VZx+IvvPvdfInrKFM39n39mAG7RZlrRLZFaXkczvmVgyaeIwluCIUIY07oJ152EKPyNKHEMPDyu+Rn3D3yL5GVCHZWacDLgmf12laBmezEssPx26fwHrvW5oqmlw03aTUn39W68eYf6hdYxuboB1p19/jmdBb6ixcDBxCc5QKiIRp6e407pq2gu+6SU8OhIDD0ti4sScTsEQ2v7IRmb5eiIRpKiIRNh0Ho6dko5eyhIU9pkWjuN58fk9AOULlcDfwpNSqZSwsoJrrx5ittOJK68e0LxibQ6buqSqOfRfIze5ZzLrL/qx+OgmjFr1ZnK7AWlf8BMSiYQKs2RiYN1qNvvlP+a3mEgWHPJi2amd5FHXZFn/qUz4t28SbZrD9y9hd2QDABYdBtGrbubrgahNaI5ESLQFK5WvMO8W+yU5x3DNDO4GPqNB2WpsGTVX4WdIpVKGrZeVUvYbO1C3TNUk72+4eJBRmxO3xaurqBFov5cieokLYwEfX9PTwxqRSMSJya6p1shT4sLzO4zebMe/VerhMcj6lzJIpVl95IkMwKhlb1YNmabwc1IjXhLPIK9ZPPzwisOmzlT/aQv81ZcP6eI2ma9R3wBoXrE2pyevzLQZQ3qQSCQM8prF7punANBS12DLyLn0rZ/xNQZF+BQegu1+D7wu+VFMtyCbRs5haOMu6Zal+K+Qm9wzkbNPbmK0bTEdqzdmxQALpf6Yqs7tT6Q4moqFS3LQ1EV+XBAEtl49gvXelXwI+8zo5gbY9zShqF7BJNd/CPtMt5VTAWhRsQ6OfSeTmdRdMJQ775/JX4uAsIQSzI/YHV7P7psnKaZbkItWa5R61kxfT3xun8Gp7yQMftCaCQ8Pp9gMfWLixfJjK/qbY9rOMMn14VERNF48mnipBM/BNkotnL38/J7eq2woV7A43uMXJ2mTm7h9Ke7nEjuC0ivgpgwW3q4cun8Rj0HWdKzeRH48WhxNaycTrr+RdcgXzpufk5PdqFM6e3eXrvP3ZcIOB7mn7bgWPVk9dHq2xhAvicf9rDez/dYQGRuNRfvBzOo2Gr08Omlf/B8kN7lnEk8/vaHPahsqFynNrrF2Sqk8dnebyovP78mrmYcncxPLCrfeBmC204kLz+/SqFwN9hotTranOS4ujtLTewBQVLcA/larfzlHWbZeOsTQTfOTHFs7aDpjWv+6MLb/9jlm+nqipa7B7Rmb0VBTfPS44ZIf9kc3Mr5lL6a2HyQ//r8FQ7n7w5dLo3LVuWqz/pfrpVIp9exHEB4TiVmb/oxv1UvhGL7LIcdLJfhNcKJQ3nyATNa3paMR0oSZi7qKGrdnbKZGyawRlHI/uwfX07uY0m4gxq0TO5EsvZfjfGIHAgKqKio49DHD/IfPKjt4/ukdrZyM+RAuW9upVaIi/lars33R9syTG5jtdOb+++d0rN6Y5f2n/jK7+dvITe6ZQEhkGN1XWqAiUsFvghP5tXXTvugnZuz34NCDS6iKVHgx3xtVVVW+RIQx64Annuf3UUgnH2uH2jKqmX6KnSKFrDojFaSoq6jx0eFwRn8sQLa7NJ91R35UIKpZvAL352xP9vwH71/Qb810VEQqnJniTvF8hRR+5tknNxm/dTEd/mmE20BLRCIRiw5vYIZv4oYfbXVNQpYeQ1MzeS2YLium8OJzIG2rNsB1gIXCMcRL4jFcM4MnH99wbJIrVYuVRSwWU2ZGzyR983O6j2Wu/liF759ejj68zKRdLujXboFjX9ni8PFHV+m9ahqR4mgAutRoit8Ep2xdqJRIJHRxm8KJx7K1Dl1NbfYZL6HdP42yLQaAtyEfsfRxZdeNk5QrWBwfo8X0+t+/f10JJjlyk3sGEcfH0W+1La9Dgjg5eQUVi5RS+B7eN0+x6MhGAPwtPSmYNx+rzvkww3cVYdGRmLbpxzz9cal+adScN4hvsbJOkdCl6ZeqTY0y03vwLvST/LWqSIV494spnh8aFU5Th7HESyVsHDGbJhXT3jH5M99nQJWKlGL3uEV8DPlM+bl9kEgT6/s7xyzEsGHKm3bMdy/j+OOrlC9UghOTFTeVEASBSbucOfboCuuGzaBttQb0X2PLnoRaMkCd0lW4M2OzwvdWhAfvX2C4ZgY1S1Rg2+j5hEVF0MJxPI8/vgagVP4inLfwpELhklkax884HNuM7T4P+aKtVYehLE7FKCYriI0T43RiG3ZHNiAVBOZ2H4t1p6HkyWYRvJxMbnLPAIIgMGG7A6ef3GDzyDlK1XQffXhF/zUzAHAfaIVEkNJo8ShuvX1Cm6r1cTU0p3ap1OunQ7xm8zDoJQBXp3mhrZ36jsm0mL1/FQsSFmS/c2bKSv5NpZVOKpVS1244EbFRWHYYzPCm3RR+bkhkGPorLREhwm+CE/XshvMq5IP8/e61muM30TnVe2y45IfLqR3oampzy1a5XbArTu/C45wP1p2GUlBbFxWTZggJc5c86hq8mOtN8YJZu5vxU3gI+u4WaGto4TfBicm7XFh/SbZgraGqzpohNgxv1j1LY/iZG68f0sl1CiFRMjmJphVqcmqKW4r7L7IKv3v+TNm9jOfB7+hd91+c+02mfKHs/YL7E8hN7hnA8fhW1l08wMyuoxjapKvC10dER1DXTiYGNrhhJy6+uMeEHUspXaAoO8YswLBBhzSnlx7n
```

Comment:
In line with my comment in the community forum, I would recommend directing readers to using the general structure for parameter sweeping and `multi_proc` from the beginning, even though there's no sweeping at this point (ie, the parameter list would contain a single item).

Reply:
For some reason GitHub's main PR UI is not displaying this comment in the right context (showing lines 299-302 for me, but comment refers to line [409](https://github.com/BlockScience/cadCAD/pull/88/files/d0d29a589da2b0e0c83b6d580833458a252770b6#r384627161)